.github/workflows/create-issue.yml

```diff
@@ -0,0 +1,25 @@
+on: push
+  issues:
+    types: [opened]
```

Comment:
@danepowell This is unrelated to release. Just replaces current Zapier functionality.